adage/analyze/tests.py

```diff
@@ -1025,3 +1034,107 @@ def test_expressionvalue_big_post(self):
+                            'conditions',
+            'description':  'To assess the role of two redox-sensitive '
+                'transcriptional regulators, RoxSR and ANR, in Pseudomonas '
+                'aeruginosa under aerobic conditions, microarray analysis was '
+                'performed. Transcriptome profiles of roxSR mutant and anr '
+                'mutant aerobically grown in LB medium were determined by '
+                'Affymetrix GeneChip at both the exponential phase and early '
+                'stationary phase and compared to that of the wild type '
+                'strain. Experiment Overall Design: Pseudomonas aeruginosa '
+                'wild type (PAO1ut), roxSR mutant (ROX1), and anr mutant '
+                '(PAO6261) strains were cultivated aerobically in LB in '
+                'Erlenmeyer flasks, and total RNAs were extracted at both the '
+                'exponential phase (OD600 = 0.3) and early stationary phase '
+                '(OD600 = 1.4). The experiment was performed in duplicate '
+                'independent cultures.'
+        }
+    ]
+
+    def setUp(self):
+        haystack.connections.reload('default')
+        super(SearchIndexTestCase, self).setUp()
+        for e in self.experiments:
+            ModelsTestCase.create_test_experiment(experiment_data=e)
+        call_command('update_index', interactive=False, verbosity=0)
+
+    def testCaseInsensitive(self):
+        """
+        We should find the same results whether we're searching with a
+        mixed-case search term or all lower case.
+        This is a check that we're handing issue #123 properly.
+        """
+        respMixed = self.api_client.get(self.searchURI, data={'q': 'mexR'})
+        self.assertValidJSONResponse(respMixed)
+        respLower = self.api_client.get(self.searchURI, data={'q': 'mexr'})
+        self.assertValidJSONResponse(respLower)
+        jsonMixed = self.deserialize(respMixed)
+        jsonLower = self.deserialize(respLower)
+        self.assertEqual(
+            jsonMixed['meta']['total_count'],
+            jsonLower['meta']['total_count']
+        )
+        self.assertEqual(
+            jsonMixed['objects'],
+            jsonLower['objects']
+        )
+
+    def testSearchPA14(self):
+        """
+        A search for 'PA14' should yield experiment E-GEOD-24262
+        This is a check that we're handling Bitbucket issue #1 properly:
+        https://bitbucket.org/greenelab/adage-server/issues/1/
+        Note: this simple test assumes no other PA14 experiments are added
+        to the test data.
+        """
+        respPA14 = self.api_client.get(self.searchURI, data={'q': 'PA14'})
+        self.assertValidJSONResponse(respPA14)
+        self.assertEqual(
+            self.deserialize(respPA14)['objects'][0]['pk'],
+            'E-GEOD-24262'
+        )
+
+    def tearDown(self):
+        call_command('clear_index', interactive=False, verbosity=0)
```

Comment:
Oh wow I hadn't realized it, but we probably need to call this after we create test indexes, huh?